app/models/municipality.rb

```diff
@@ -0,0 +1,7 @@
+class Municipality < ApplicationRecord
+  belongs_to :package
+  has_many :prices
+
+  validates :name, presence: true, uniqueness: true
+  validates :price_cents, presence: true
```

Comment:
We have `has_many: prices` and we have `price_cents`. Why do we need both?

Reply:
We can eliminate the price_cents, but I've assumed the same structure as for the `Package` that `Municipality` will store the current price in `price_cents` and `has_many: prices` will store the history of price.